### DISCUSSION TITLE
Logging Val Loss w/ MMDetWandbLogger

### DISCUSSION THREAD

**Discussion by OrcunCanDeniz** [open-mmlab/mmcv#2995] [General]
I've multiple training runs. I noticed that for some of them validation losses is logged as wandb charts and for some of them wasn't. What might be causing this behaviour difference? Those runs have variations on versions of mmlab libs (one run has mmcv==2.0.0rc1, other mmcv==1.7.0; while both having same mmcv-full=1.6.0. Running `mmcv.__version__` yields 1.6.0 for both)
